server/index.js

```diff
@@ -90,17 +88,16 @@ app.post('/api/cart', (req, res, next) => {
   db.query(sql1, parameterizedArray)
     .then(results => {
       if (!results.rows.length) {
-        return next(new ClientError(`cannot find id at ${productId}`), 400);
+        next(new ClientError(`cannot find id at ${productId}`), 400);
+        process.exit(1);
```

Comment:
We don't want to `process.exit()` that will kill the server. Instead `throw new ClientError(message, status)`